README.md

```diff
@@ -91,7 +116,14 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+
+var doSomething = function (thingToDo){
+  thingToDo();
+}
+doSomething(){
+  console.log("hello");
+}
```

Comment:
Here we just want to invoke `doSomething` and pass in a function to invoke like: 

```js
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```